templates/documentation/delivery-analytics/private-videos-with-custom-players-session-retention.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: "Private videos with custom players & session retention"
+title: Private video session retention on custom players
+slug: private-videos-with-custom-players-session-retention
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)

Reply:
Done ✅